### ISSUE TITLE
🐛 nmtui window from waybar doesn't scale

### ISSUE THREAD

**Issue by theRoboxx** [manjaro-sway/manjaro-sway#489]
### branch

stable

### Architecture

x86_64

### Description

If I click on the nmtui button from waybar it opens the program in a seperate window but in some windows the resolution doesn't scale. E.g. See here:
![swappy-20230331_161153](https://user-images.githubusercontent.com/63428105/229145823-ad44a7bc-c2a3-4f18-887e-32d3e554a191.png)

### Reproduction steps

```bash
- Click on the nmtui button from waybar
- Edit a new connection
- <Add> 
OR
- <Edit...>
```

**Comment by ishaanbhimwal** [comment]
This is a nmtui issue and not related to wayland, sway or foot. Maybe try toggling the window from floating mode with <kbd>Mod+Shift+Space</kbd>.

* https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/issues/576

**Comment by theRoboxx** [comment]
> This is a nmtui issue and not related to wayland, sway or foot. Maybe try toggling the window from floating mode with Mod+Shift+Space.
> 
> * https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/issues/576

**tl;dr: Fixed it myself, while writing this issue. Solution at bottom.** 

Unfortunately toggling the window from floating mode with `Mod+Shift+Space` still shows the buggy text (see 1.1). 
This library [libnewt](https://pagure.io/newt) seems to cause the main problem overall when changing the terminal size.
Now, I tested another approach: Open it in the terminal with `nmtui` works far better than toggling the floating window with `Mod+Shift+Space`, so there seems to be a difference (see 2.1).

1.1: Toggled floating window with `Mod+Shift+Space` while other widows are open:
![swappy-20230408_154035](https://user-images.githubusercontent.com/63428105/230724343-a9fd58e3-2efb-4605-b81f-f3c97f093ac1.png)

1.2: Toggled floating window with `Mod+Shift+Space` and `Mod+f` for fullscreen:
![swappy-20230408_150703](https://user-images.githubusercontent.com/63428105/230723113-70a91e8b-6926-41f5-a3cc-4305145a5aad.png)

2.1: Opened `nmtui` in terminal while other widows are open:
![swappy-20230408_150818](https://user-images.githubusercontent.com/63428105/230723131-ee4d1e86-cf9a-46f6-9afd-fad6fa791ec8.png)

2.2: Opened `nmtui` in terminal in fullscreen mode:
![swappy-20230408_150843](https://user-images.githubusercontent.com/63428105/230723146-2ee1169e-be54-4d05-b8e5-23773c2a5c8a.png)

**FIX**:

Add slightly bigger initial floating terminal size for the nmtui command to my custom definitions `~/.config/sway/config.d/definitions.conf`:
``` 
set $term_float_for_nmtui footclient -a floating_shell --window-size-pixels=900x600
```
and added the new variable to my `~/.config/waybar/config.jsonc`:
```
    "network": {
        "interval": 5,
        "format-wifi": " ",
        "format-ethernet": "",
        "format-disconnected": "睊",
        "tooltip-format": "{ifname} ({essid}): {ipaddr}",
        "on-click": "swaymsg exec \\$term_float_for_nmtui nmtui"
    },
```

**Comment by theRoboxx** [comment]
Maybe increase the default floating window size of nmtui for everyone. Or does it depend on the screen size and resolution(1920x1080)?

**Comment by boredland** [comment]
> Maybe increase the default floating window size of nmtui for everyone. Or does it depend on the screen size and resolution(1920x1080)?

Lets give an increase to 900 a try for all floating shells. I don't assume it will break for too many people.
